content/docs/1_guide/25_updates/1_update-to-v4/guide.txt

```diff
@@ -23,17 +23,17 @@ Kirby 4 requires PHP 8.1, 8.2 or 8.3. You can check your current PHP version in
 
 If you see **PHP 8.1.\*, PHP 8.2.\* or PHP 8.3.\***, you are already good to go.
 
-If you see **PHP 7.\* or PHP 8.0.\***, you need to update your server. Most hosting providers allow changing the PHP version from the admin backend. We recommend to switch to PHP 8.2 because this version is already compatible with Kirby 3.9 and will (link: https://www.php.net/supported-versions.php text: receive security updates for two more years), so you won't have to do another PHP migration until then.
```

Comment:
The references to Kirby 3 and PHP 8.1 in this file are intended. It's a migration guide after all.